frontend/workflows/ec2/src/index.tsx

```diff
@@ -13,7 +13,15 @@ interface ConfirmConfigProps {
   notes?: NoteConfig[];
 }
 
-export interface WorkflowProps extends BaseWorkflowProps, ResolverConfigProps, ConfirmConfigProps {}
+interface WizardFeedbackProps {
```

Comment:
Is it possible to export and extend this interface from @clutch-sh/core instead of redefining it in each workflow separately?

Reply:
ah good idea, thx for the suggestion!